addons/binding/org.openhab.binding.nuki/src/main/java/org/openhab/binding/nuki/handler/NukiSmartLockHandler.java

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.nuki.handler;
+
+import org.eclipse.smarthome.config.core.Configuration;
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.RefreshType;
+import org.eclipse.smarthome.core.types.State;
+import org.openhab.binding.nuki.NukiBindingConstants;
+import org.openhab.binding.nuki.dataexchange.BridgeLockActionResponse;
+import org.openhab.binding.nuki.dataexchange.BridgeLockStateResponse;
+import org.openhab.binding.nuki.dataexchange.NukiHttpClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link NukiSmartLockHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Markus Katter - Initial contribution
+ */
+public class NukiSmartLockHandler extends BaseThingHandler {
+
+    private final static Logger logger = LoggerFactory.getLogger(NukiSmartLockHandler.class);
```

Comment:
Logger should NOT be static according to the guidelines. [[source](http://docs.openhab.org/developers/development/guidelines.html#e-logging)]